script/Arcueid/Day 10/10_00_ARC10_4.txt

```diff
@@ -779,7 +779,7 @@ is unknown to me."
 [offset:25143]{
 -- Page 350, Offset 25143.
 -- 「……？」
-"...Hm?"
+"Hm...?"
```

Comment:
Ellipses at the start works better. It's not a lingering thought but a '...wait what?' moment.

Reply:
I hate leading ellipsis
I hate leading ellipsis
I hate leading ellipsis
I hate leading ellipsis
I hate leading ellipsis